tox.ini

```diff
@@ -33,7 +34,7 @@ install_command =
 commands = {posargs}
 
 [testenv:pep8]
-commands = pep8 --exclude=build,dist,.eggs,utils,migrations,.tox,doc/source/conf.py,node_modules/casperjs,node_modules/grunt-casper .
+commands = pep8 --exclude=build,dist,.eggs,utils,migrations,.tox,doc/source/conf.py,node_modules/casperjs,node_modules/grunt-casper,docker/Dockerfiles/goldstone-app/goldstone-server .
```

Comment:
Did you do this before the app container was recreated?  There is code in the entrypoint that touches this file in order to avoid just this situation.  If you haven't recreated the container, maybe you want to hold off until I address the issues that @lexjacobs raised below.